deps/wazuh_testing/wazuh_testing/tools/macos_log/macos_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+def compose_settings(__type="", __level="", __predicate="", __is_sierra=False):
+    """
+    This function replicates how the command 'log' will be called from the Wazuh agent given the query parameters
+    """
+
+    settings_str = ""
+
+    if (__is_sierra):
+        settings_str = "/usr/bin/script -q /dev/null "
+
+    settings_str = settings_str + "/usr/bin/log stream --style syslog "
+
+    if (__type):
+        __type = __type.replace(" ", "")
+        for t in __type.split(","):
+            settings_str = settings_str + "--type " + t + " "
+
+    if (__level):
+        __level = __level.replace(" ", "")
```

Comment:
Why this is required? Maybe it should be desirable to pass as arguments a properly formatted level.